crates/apps/src/lib/cli/wallet.rs

```diff
@@ -1243,6 +1247,25 @@ fn key_export(
         })
 }
 
+/// Convert a consensus key to tendermint validator key in json format
+fn key_convert(
+    ctx: Context,
+    io: &impl Io,
+    args::KeyConvert { alias }: args::KeyConvert,
+) {
+    let alias = alias.to_lowercase();
+    let mut wallet = load_wallet(ctx);
+    let sk = wallet.find_secret_key(&alias, None);
+    let key: serde_json::Value = validator_key_to_json(&sk.unwrap()).unwrap();
+    let file_name = format!("priv_validator_key.json_{}", alias);
```

Comment:
Is it intended that the file will looks like `priv_validator_key.json_alias` rather than `priv_validator_key_alias.json` or smth similar?

Reply:
Should I modify the code about that?